### PR TITLE
Adds formula for cpyrit opencl.

### PR DESCRIPTION
**Pull request by timemaster5**
Opencl support for pyrit

### CODE REVIEW

File: pyrit.rb [hunk 1]
```diff
@@ -1,16 +1,21 @@
-require 'formula'
-
 class Pyrit < Formula
-  homepage 'https://code.google.com/p/pyrit/'
-  url 'https://pyrit.googlecode.com/files/pyrit-0.4.0.tar.gz'
-  sha1 '3041c0d593c21df88f41416be872eca0997cf22b'
+  desc "Pyrit allows to create massive databases, pre-computing part of the WPA/WPA2-PSK authentication phase in a space-time-tradeoff. Exploiting the computational power of Many-Core- and other platforms through ATI-Stream, Nvidia CUDA, OpenCL and VIA Padlock, it is currently by far the most powerful attack against one of the world's most used security-protocols."
```
Comment (tdsmith) [inline pyrit.rb]: This should be ≤ 80 characters, including a leading "pyrit: ".

File: cpyrit-opencl.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  url "https://pyrit.googlecode.com/files/cpyrit-opencl-0.4.0.tar.gz"
+  sha256 "aac593bce3f00ea7fd3d558083dbd7e168332a92736e51a621a0459d1bc042fa"
+
+  depends_on :python
+  depends_on "libdnet"
+  depends_on "pyrit"
+
+  def install
+    system "python", "setup.py", "build"
+    system "python", "setup.py", "install", "--prefix=#{prefix}"
```
Comment (tdsmith) [inline cpyrit-opencl.rb]: Can both of these lines be replaced by `system "python", *Language::Python.setup_install_args(prefix)`?

File: cpyrit-opencl.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+class CpyritOpencl < Formula
+  desc "GPU-accelerated attack against WPA-PSK authentication"
+  homepage "https://code.google.com/p/pyrit/"
+  url "https://pyrit.googlecode.com/files/cpyrit-opencl-0.4.0.tar.gz"
+  sha256 "aac593bce3f00ea7fd3d558083dbd7e168332a92736e51a621a0459d1bc042fa"
+
+  depends_on :python
```
Comment (tdsmith) [inline cpyrit-opencl.rb]: Does this depend on :python? If we drop the :python dependency, we should be able to build this against system python and get a module that works with Homebrew python. (The same change can be made to pyrit.)

File: cpyrit-opencl.rb (hunk 2 above)
Comment (tdsmith) [inline cpyrit-opencl.rb]: (The same change can be made to pyrit.)

File: pyrit.rb [hunk 4]
```diff
@@ -1,16 +1,21 @@
...
+  depends_on "scapy"
 
   def install
       system "python", "setup.py", "build"
       system "python", "setup.py", "install", "--prefix=#{prefix}"
   end
+
+  test do
+    Language::Python.each_python(build) do |python, version|
+      system python, "-c", "import pyrit_cli
```
Comment (tdsmith) [inline pyrit.rb]: Missing a trailing "

**Comment by timemaster5** [comment]
I fixes everything you wrote..Thanks tdsmith

**Comment by timemaster5** [comment]
uhh.. I am to small to manage CI build issues :( can anyone help ?

**Comment by tdsmith** [comment]
Pulled with a minor fix to the test; thanks for contributing! In the future please only touch one formula per commit; I had to rebase all of these.

**Pull request closed without merging**